rpcs3/Emu/RSX/VK/vkutils/swapchain.hpp

```diff
@@ -710,10 +710,23 @@ namespace vk
 
 	#ifdef _WIN32
 			VkSurfaceFullScreenExclusiveInfoEXT full_screen_exclusive_info = {};
-			if (should_disable_exclusive_full_screen)
+			if (should_specify_exclusive_full_screen_mode)
 			{
 				full_screen_exclusive_info.sType               = VK_STRUCTURE_TYPE_SURFACE_FULL_SCREEN_EXCLUSIVE_INFO_EXT;
-				full_screen_exclusive_info.fullScreenExclusive = VK_FULL_SCREEN_EXCLUSIVE_DISALLOWED_EXT;
+				switch (g_cfg.video.vk.exclusive_fullscreen_mode)
+				{
+				case vk_exclusive_fs_mode::enable:
+					full_screen_exclusive_info.fullScreenExclusive = VK_FULL_SCREEN_EXCLUSIVE_ALLOWED_EXT;
+					break;
+				case vk_exclusive_fs_mode::disable:
+					full_screen_exclusive_info.fullScreenExclusive = VK_FULL_SCREEN_EXCLUSIVE_DISALLOWED_EXT;
+					break;
+				case vk_exclusive_fs_mode::unspecified:
+				default:
+					rsx_log.error("Invalid exclusive fs mode");

```

Comment:
why is this logging an error? that makes no sense. The default case is obsolete anyway

Reply:
It should not be possible to enter into that block with unspecified  fs because that would mean we have skipped the capability test. Not sure why I went with errorz i should have made it throw I guess.